src/pudl/metadata/enums.py

```diff
@@ -1,97 +1,39 @@
 """Enumerations of valid field values."""
 
-US_STATES: dict[str, str] = {
-    "AK": "Alaska",
-    "AL": "Alabama",
-    "AR": "Arkansas",
-    "AZ": "Arizona",
-    "CA": "California",
-    "CO": "Colorado",
-    "CT": "Connecticut",
-    "DE": "Delaware",
-    "FL": "Florida",
-    "GA": "Georgia",
-    "HI": "Hawaii",
-    "IA": "Iowa",
-    "ID": "Idaho",
-    "IL": "Illinois",
-    "IN": "Indiana",
-    "KS": "Kansas",
-    "KY": "Kentucky",
-    "LA": "Louisiana",
-    "MA": "Massachusetts",
-    "MD": "Maryland",
-    "ME": "Maine",
-    "MI": "Michigan",
-    "MN": "Minnesota",
-    "MO": "Missouri",
-    "MS": "Mississippi",
-    "MT": "Montana",
-    "NC": "North Carolina",
-    "ND": "North Dakota",
-    "NE": "Nebraska",
-    "NH": "New Hampshire",
-    "NJ": "New Jersey",
-    "NM": "New Mexico",
-    "NV": "Nevada",
-    "NY": "New York",
-    "OH": "Ohio",
-    "OK": "Oklahoma",
-    "OR": "Oregon",
-    "PA": "Pennsylvania",
-    "RI": "Rhode Island",
-    "SC": "South Carolina",
-    "SD": "South Dakota",
-    "TN": "Tennessee",
-    "TX": "Texas",
-    "UT": "Utah",
-    "VA": "Virginia",
-    "VT": "Vermont",
-    "WA": "Washington",
-    "WI": "Wisconsin",
-    "WV": "West Virginia",
-    "WY": "Wyoming",
+from pudl.metadata.dfs import POLITICAL_SUBDIVISIONS
+
+COUNTRY_CODES_ISO3166: set[str] = set(POLITICAL_SUBDIVISIONS.country_code)
+SUBDIVISION_CODES_ISO3166: set[str] = set(POLITICAL_SUBDIVISIONS.subdivision_code)
+EPACEMS_STATES: set[str] = set(
+    POLITICAL_SUBDIVISIONS.loc[
+        POLITICAL_SUBDIVISIONS.is_epacems_state, "subdivision_code"
+    ]
+)
+DIVISION_CODES_US_CENSUS: set[str] = set(
+    POLITICAL_SUBDIVISIONS.division_code_us_census.dropna()
+)
+
+APPROXIMATE_TIMEZONES: dict[str, str] = {
```

Comment:
None of these are tables, and they don't get loaded into the DB. They're mainly referred to during the ETL to provide ENUM constraints on various fields, or in some cases they're used in the transform functions. Since there's no DB to refer to at that point, they need to be read out of the stored dataframe.

It may make sense to turn some/most of the ENUMs into FK constraints across the board (there's a lot of places that refer to states) but that will probably require a bunch of additional data cleaning. If/when we're incrementally building up the DB over the course of the ETL instead of compiling a big dictionary of dataframes that all get loaded at the end, we could move to reading the data out of the DB instead of referring to the constants.